assets/js/wfparameters/RenameEntries.js

```diff
@@ -0,0 +1,194 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import WorkBenchAPI from '../WorkbenchAPI'
+
+var api = WorkBenchAPI();
+export function mockAPI(mock_api) {
+    api = mock_api;
+}
+
+export class RenameEntry extends React.Component {
+    static propTypes = {
+        colname: PropTypes.string.isRequired,
+        newColname: PropTypes.string.isRequired,
+        onColRename: PropTypes.func.isRequired,
+        onEntryDelete: PropTypes.func.isRequired,
+    };
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            inputValue: this.props.newColname
+        };
+
+        this.handleChange = this.handleChange.bind(this);
+        this.handleKeyPress = this.handleKeyPress.bind(this);
+        this.handleBlur = this.handleBlur.bind(this);
+        this.handleFocus = this.handleFocus.bind(this);
+        this.handleDelete = this.handleDelete.bind(this);
```

Comment:
Babel gives us a fancier way of doing this. Instead of a whole bunch of `.bind` methods in the ctor, just declare bound methods this way:

```javascript
handleDelete = (ev) => {
  ...
}
```

No need to change this code. But I think the fancy way should be our standard in the future.